.docker/web/Dockerfile

```diff
@@ -4,9 +4,6 @@ COPY .docker/web/vhost.conf /etc/apache2/sites-available/000-default.conf
 
 WORKDIR /srv/app
 
-RUN mkdir -p /srv/app/storage/code
-RUN chmod -R ug+rwx /srv/app/storage/code && chown -R www-data:www-data /srv/app && a2enmod rewrite
```

Comment:
Just checking if this wasn't accidental:

L07
Can it be deleted because tempnam already takes care of it?
$filename = tempnam(__DIR__ . '/../storage/code', $file_key);

L08:
Parts 1 and 2 can be deleted, because of @chmod($filename, 0666);?
Part 3 (rewrite): Can this be deleted because rewrite is always enabled in this image or we're not using it?

Reply:
Yeah, this was intentional, but i will revert the change.

We don't use tempfles anymore (see https://github.com/dev-community-de/code-formatter/pull/27), but having a storage folder doesn't hurt anyone (we could probably use it for caching and so on in the future).